### ISSUE TITLE
[Channel Request] - LightAMV's Server (100IV, > 90IV)

### ISSUE THREAD

**Issue by LightAMV** [5andr0/PogoLocationFeeder#306]
https://discord.gg/Xws2K
#100iv
#90iv

Small group dedicated to NecroBot, PogoLocationFeeder, and PokeSniper2.
Rules are respected.

**Comment by vfuyong** [comment]
Can you share the json config to add this channel?

**Comment by LightAMV** [comment]
@vfuyong I was told that you cannot add your own custom channel. When I did try to add it, it ends up crashing.

**Comment by vfuyong** [comment]
I don't think that is the case. I added it successfully, but I am not too sure on the format of the json. e.g. what is id, server and name.

**Comment by LightAMV** [comment]
@vfuyong I managed to add it, but it doesn't show up at all, or the logs.

**Comment by BehindU** [comment]
I've talked to a dev on the discord channel.. Adding your own discords channel doesn't work, only devs can do that. Just wait for one to review this and it'll be done :)

**Comment by BehindU** [comment]
https://discord.gg/N3tup

There you go
